.github/workflows/deploy.yml

```
@@ -0,0 +1,59 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
+      - master
+      - develop
+      - development
```

Based on our convention, now you just need 2 branches
```suggestion
      - main
      - develop
```